packages/core/src/database/models/learnAndEarn/learnAndEarnLevel.ts

```diff
@@ -20,6 +20,11 @@ export class LearnAndEarnLevelModel extends Model<
     public asset!: string;
     public adminUserId!: number;
     public status!: string;
+    public rules!: {
+        countries: string[];
+        roles: string[];
+        limitUsers: number;
+    };
```

Comment:
If it was a simple read field, I'd be fine with this. But we have to apply filters here. I think this can get way too complex. I'd suggest, each one of these is a separate column.